### DISCUSSION TITLE
ElastoDyn number of blades

### DISCUSSION THREAD

**Discussion by peterfakhry** [OpenFAST/openfast#1151] [Q&A] [answered]
Hello there, i have three questions. Can `ElastoDyn `solve for all the three blades instead of just the first one because `BeamDyn `needs a very small sampling time to converge. The second question what is the distance `x `shown in the figure defined in the `FAST `files. Third question why always the in plane deflection of blade is plotted in the -ve direction?
Thank you
![image](https://user-images.githubusercontent.com/40067011/173817891-dc3049bf-0594-4ff0-bf36-9591b8f76b5f.png)

**Comment by jjonkman** [comment, marked as answer]
Dear @peterfakhry,

Here are my answers to your questions:
1) Yes, ElastoDyn can be used to model wind turbines with 1, 2, or 3 blades through input `NumBl`.
2. In ElastoDyn, the blade tip to tower-centerline clearance in the undeflected condition when the blade is pointing downward can be computed as: `ABS( OverHang*COS(ShftTilt) + TipRad*SIN(ShftTilt+PreCone) )`
3. ElastoDyn uses the IEC-style coordinate system where the y-axis of the blade coordinate system points towards the trailing edge at the zero-twist blade station.

Best regards,

**Reply by peterfakhry** [reply]
thank you sir for this clarification. Regarding the first question, i mean i only get the response of the first blade in the `.out` file however i want the three blades response 
thank you

**Reply by jjonkman** [reply]
You can include outputs from all three blades in the `OutList` at the bottom of the ElastoDyn input file.

Best regards,

**Reply by peterfakhry** [reply]
Thanks  a lot Professor
